layers.py

```diff
@@ -311,7 +311,10 @@ def forward(
             dense_xyz_norm,
             sparse_xyz=None,
             sparse_xyz_norm=None,
-            vi_features=None):
+            vi_features=None,
+            inv_neighbors=None,
+            inv_k=None,
+            inv_idx=None):
```

Comment:
Comment of the function should include these 4 variables along with their sizes as well.